candle-nn/src/rnn.rs

```diff
@@ -128,25 +137,29 @@ pub fn lstm(
     vb: crate::VarBuilder,
 ) -> Result<LSTM> {
     let layer_idx = config.layer_idx;
+    let direction_str = match config.direction {
+        Direction::Forward => String::from(""),
```

Comment:
Why use `String` rather than `&'static str` here?

Reply:
Thanks for pointing that out. I'm still new to Rust, so wasn't quite sure about using `String` vs just having `&'static str'. But I see its probably more efficient to use `&'static str`. I updated the code per your suggestion.